master/security/service-accounts.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Use service accounts in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow/deny traffic for Kubernetes services. 
+
+### Value
+
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the network security can have RBAC permissions to:
```

Comment:
> For example, the network security can have RBAC permissions to

"the network security" doesn't make sense here I don't think. What is this sentence meant to be saying?

Reply:
I think I understand the intention here now. How about something like this?

> Kubernetes RBAC allows you to control which users are allowed to create and use service accounts. Combined with Calico network policy, this allows for a security boundary between entities provisioning service accounts and those who are using them.

> For example, permission to provision new service accounts can be restricted using Kubernetes RBAC to only the network security team, who can then write high-priority network policies which reference those service accounts.

Reply:
I added this wording with some minor tweaks for clarity.

Reply:
Fixed